app/views/lists/_list.html.slim

```diff
@@ -0,0 +1,8 @@
+tr
+  td
+  td
+    span
+      = list.name
+  td.eye-btn.text-center = link_to list
+    button.btn.btn-outline-dark
+      i.fas.fa-eye
```

Comment:
missing the final blank line at the end of the file

Reply:
done